sorbet/rbi/shims/type_aliases.rbi

```diff
@@ -0,0 +1,21 @@
+# typed: true
+
+AnyConstant = T.type_alias do
```

Comment:
Not having this at runtime will make the signatures raise when we try to generate RBIs for the indexer with Tapioca